logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -141,6 +165,57 @@ class ConversationDataSource(
     override suspend fun deleteMember(conversationID: QualifiedIDEntity, userID: QualifiedIDEntity): Either<CoreFailure, Unit> =
         wrapStorageRequest { conversationDAO.deleteMemberByQualifiedID(conversationID, userID) }
 
+    override suspend fun createGroupConversation(name: String, members: List<Member>, options: ConverationOptions): Either<CoreFailure, Conversation> = suspending {
+        wrapApiRequest {
+            conversationApi.createNewConversation(
+                CreateConversationRequest(
+                    if (options.protocol == ConverationOptions.Protocol.PROTEUS) members.map { idMapper.toApiModel(it.id) } else emptyList(),
+                    name,
+                    options.access.toList().map(conversationMapper::toApiModel),
+                    options.accessRole.toList().map(conversationMapper::toApiModel),
+                    userRepository.getSelfUser().firstOrNull()?.team?.let { ConvTeamInfo(false, it) },
+                    null,
```

Comment:
I think since we have some values that are not clear to the assigned parameter, it would be better to use the named arguments to make the code more readable.